api/v1/helpers/verbs/heartbeatUtils.js

```diff
@@ -11,20 +11,15 @@
  */
 'use strict'; // eslint-disable-line strict
 
-const collectorMap = {};
-const propsToSend = [
-  'aspects', 'collectors', 'connection', 'context',
-  'description', 'generatorTemplate', 'helpEmail', 'helpUrl', 'id', 'name',
-  'subjectQuery', 'subjects', 'tags',
-];
-const ADDED = 1;
-const DELETED = 2;
-const UPDATED = 3;
-const NOCHANGE = 4;
+const redisClient = require('../../../../cache/redisCache').client.heartbeat;
+const ADDED = 'added';
+const DELETED = 'deleted';
+const UPDATED = 'updated';
+const NOCHANGE = 0;
```

Comment:
When we inspect the redis instance, the key of the unchanged generators (for collector1)will look like `heartbeat::generatorChanges::collector1::0`. Can we change it to something more meaningful?

Reply:
Redis will only be updated when there is a change. NOCHANGE is only there to make the logic clearer.

Reply:
Ok. So we will never see a key like `heartbeat::generatorChanges::collector1::0` in redis, correct?

Reply:
correct.